docs/source/user_manual/modules_and_classes.rst

```diff
@@ -197,36 +238,41 @@ first in Chaco, and then moved into Enable.
 
 *Subclasses*:
 
+* :class:`~.BetterZoom`
+
+  * :class:`~.BetterSelectingZoom`
+  
+    * :class:`~.RectZoomTool`
+    * :class:`~.TrackingZoom`
+
 * :class:`~.BroadcasterTool`: Keeps a list of other tools, and broadcasts
   events it receives to all those tools.
 * :class:`~.DataPrinter`: Prints the data-space position of the point
   under the cursor.
 * :class:`enable.tools.drag_tool.DragTool`: Enable base class
   for tools that do dragging.
 
+  * :class:`~.DataLabelTool`
+  * :class:`~.DragZoom`
+  * :class:`~.LegendTool`
+
+    * :class:`~.LegendHighlighter`
+
   * :class:`~chaco.tools.move_tool.MoveTool`
+  * :class:`chaco.tools.pan_tool2.PanTool`
   * :class:`enable.tools.resize_tool.ResizeTool`
   * :class:`enable.tools.viewport_pan_tool.ViewportPanTool`
-
-* :class:`~.chaco.tools.drag_tool.DragTool`: Chaco base class
-  for tools that do dragging.
-
   * :class:`~chaco.tools.cursor_tool.BaseCursorTool`
 
     * :class:`~chaco.tools.cursor_tool.CursorTool1D`
     * :class:`~chaco.tools.cursor_tool.CursorTool2D`
 
-  * :class:`~.DataLabelTool`
-  * :class:`~.DragZoom`
-  * :class:`~.LegendTool`
-  * :class:`~.MoveTool`
-
 * :class:`~.DrawPointsTool`
 * :class:`~.HighlightTool`
 * :class:`enable.tools.hover_tool.HoverTool`
 * :class:`~.ImageInspectorTool`
 * :class:`~.LineInspector`
-* :class:`~.PanTool`
+* :class:`chaco.tools.pan_tool.PanTool`
```

Comment:
tangential question - this and `pan_tool2` look similar but im not sure what their differences are - and why

Reply:
I agree, I've moved #438 into the project for the quarter